dll/win32/kernel32/kernel32_vista/CMakeLists.txt

```diff
@@ -13,7 +13,13 @@ list(APPEND SOURCE
     sync.c
     vista.c)
 
+if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
```

Comment:
Could/Should this check be removed and [kernel32_vista.spec](https://github.com/reactos/reactos/blob/master/dll/win32/kernel32/kernel32_vista/kernel32_vista.spec) updated?

Reply:
I'll make it unconditional, but I'll not change the spec, since we don't need those exports (yet).